schema/enums/StockPlanCancellationBehaviorType.schema.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StockPlanCancellationBehaviorType.schema.json",
+  "title": "Enum - Stock Plan Cancellation Behavior Type",
+  "description": "For a given stock plan, what is the default rule for what happens to the shares reserved for a Plan Security after it's cancelled.",
+  "type": "string",
+  "enum": ["RETIRE", "RETURN_TO_POOL", "CAPITAL_STOCK"],
```

Comment:
Add an option for per_security

Reply:
We should also update the validation meta issue with this rule - what implications per_security has.